src/main/java/com/google/error_analyzer/backend/LogDaoHelper.java

```diff
@@ -60,4 +60,9 @@
         }
         return searchResult;
     }
+
+    //returns index name which stores the error logs
+    public static String getErrorIndexName (String fileName) {
+        return fileName.concat("error");
```

Comment:
Unit tests?

Reply:
I added unit test for this in BooleanQuery PR